app/controllers/HelpAndContactController.scala

```diff
@@ -88,15 +88,16 @@ class HelpAndContactController @Inject()(
             case 200 => Redirect(successRedirect).withSession(request.session + (TICKET_ID -> resp.body))
             case 400 => BadRequest(failedValidationResponseContent(Html(resp.body)))
             case 500 => {
-              Logger.warn("submit contact form internal error 500, " + resp.body)
+              Logger.error(
+                s"[HelpAndContactController][submitContactHmrcForm] Submit contact form internal error 500, ${resp.body}")
               InternalServerError(Html(resp.body))
             }
             case status => throw new Exception(s"Unexpected status code from contact HMRC form: $status")
```

Comment:
I'd like an error log for the unexpected status. We can at least see what was in the body?